include/h3api/H3Dialogs/H3CustomDialog.cpp

```diff
@@ -41,6 +41,12 @@ namespace h3
 	}
 	_H3API_ H3Dlg::~H3Dlg()
 	{
+		THISCALL_1(VOID, 0x5FFAD0, this); // destroy all dlg items
+		if (this->background)
+		{
+			this->background->Destroy(); // destroy background
```

Comment:
No, the background is added to the controls vector, which is being deleted by the call to 0x5FFAD0.

This would be deleting it a second time and asking for trouble.

Reply:
Nvm, thought we were talking about the widget. This is correct.